sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert (year / week).evalf(15) == Float(52.17745571428571)
```

Comment:
Also I'm well aware of the floating point comparisons. If you looked at the history of the thread you would see that I suggested a correct way to do that.

Secondly, all this PR then needs is quotes around the float value: `Float('52.17745571428571')` and there is no issue. I missed that and notice that I wrote "I think" when I suggested it.

Thanks for finding the error.